db/seeds/index.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Locations = require('./data/locations');
+const Movies    = require('./data/movies');
 
 exports.seed = function (Knex) {
```

Comment:
ah yeah it's because of the [`no-shadow`](http://eslint.org/docs/rules/no-shadow) rule. the template that knex gives us is a bit different and goes against some of the default lint rules we have. thats why we have the [`lob/migrations`](https://github.com/lob/eslint-config-lob/blob/b9ce3d033b79ae15f1e20d76a0a8398cba54c856/migrations.js) config to allow these differences. you just need to copy the [`.eslintrc`](https://github.com/vinay-pad/sfmovies/blob/874bd97a9844b9b09377534c28a70539602e5d6e/db/migrations/.eslintrc) that you use for the migrations into the `db/seeds` directory